### PR TITLE
Create vsix that contains all roslyn dependencies

### PR DESCRIPTION
**Pull request by jmarolf**
@jasonmalinowski @jaredpar

cc: @dotnet/roslyn-ide @dotnet/roslyn-infrastructure

### CODE REVIEW

File: src/Deployment/Roslyn.csproj [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <RootNamespace>Roslyn</RootNamespace>
+    <AssemblyName>RoslynDeployment</AssemblyName>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <GeneratePkgDefFile>false</GeneratePkgDefFile>
+    <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
+    <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
+    <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
```
Comment (jasonmalinowski) [inline src/Deployment/Roslyn.csproj]: We set this in common targets.

**Comment by jasonmalinowski** [comment]
:+1: if it works. @dotnet-bot, test vsi please.

**Comment by jaredpar** [comment]
:+1:

**Comment by basoundr** [comment]
Should this contain the telemetry vsix?

**Comment by jmarolf** [comment]
@balajikris we could do that, in the future.  I would like to have something that install the bare minimum set of dependencies.  This also doesn't install the Roslyn diagnostics window or the test setup vsix

**Comment by jmarolf** [comment]
test vsi please.

**Comment by jmarolf** [comment]
retest prtest/win/vsi/p2 please

**Pull request merged**